src/Files.App/Views/ShellPanesPage.xaml.cs

```diff
@@ -246,14 +246,21 @@ public ShellPanesPage()
 
 			// Set default values
 			ActivePane = GetPane(0);
-			_WindowIsCompact = MainWindow.Instance.Bounds.Width <= Constants.UI.MultiplePaneWidthThreshold;
+
+			try
+			{
+				_WindowIsCompact = MainWindow.Instance.Bounds.Width <= Constants.UI.MultiplePaneWidthThreshold;
```

Comment:
Can you leave a comment
// Ignore a chance of the main window being already disposed (#xxx)

Reply:
Thanks